code/modules/research/rdconsole.dm

```diff
@@ -150,7 +150,7 @@ doesn't have toxins access.
 		return FALSE
 	var/price = TN.get_price(stored_research)
 	if(stored_research.research_points >= price)
-		investigate_log("[key_name_admin(user)] researched [id]([price]) on techweb id [stored_research.id].")
+		investigate_log("[key_name_admin(user)] researched [id]([price]) on techweb id [stored_research.id].", INVESTIGATE_RESEARCH)
 		if(stored_research == SSresearch.science_tech)
 			if(stored_research.researched_nodes.len < 30)
 				SSblackbox.record_feedback("tally", "science_techweb_unlock_first_thirty", 1, "[id]")

```

Comment:
Answer me nerd https://github.com/tgstation/tgstation/pull/31026#discussion_r151854757

Reply:
it's separate because 30 nodes is not the same as all of them, as you can reasonably get all of them unless the round is shorter than 90 minutes, but the first 30 shows what people are rushing.
The entire feedback system for techwebs is horrid and I'll be thinking about ways to improve it over the next few days.

Reply:
I was thinking I'd make it track the time at which each node is unlocked.

Reply:
pls do that :P